src/WebJobs.Script/Binding/HttpBinding.cs

```diff
@@ -194,7 +186,7 @@ internal static HttpContent CreateResultContent(object content, string mediaType
             string stringContent;
             if (content is ExpandoObject)
             {
-                stringContent = Utility.ToJson((ExpandoObject)content);
+                stringContent = Utility.ToJson((ExpandoObject)content, Formatting.None);
```

Comment:
Making this change for wire optimization, as well as to be consistent with WebApi default ObjectContent formatting of ExpandoObjects (below).